### PR TITLE
Updated Metrics and Messages guides according to agreed style guidelines

### PR DESCRIPTION
**Pull request by AedinC**
Made the following updates in this preview of the [Messages](http://file.ams.redhat.com/aecollin/JK-9743/Messages/README.html) guide:

- Broke 2nd sentence into list for improved readability
- Amended one instance of 'console' to 'web console'
- Ensured full name of product used on first occurrence in each module
- Replaced 'drop-down menu- with 'list'
- Removed italics for guide links
- Removed references to 'offset' as default value and amended this default value to 'latest messages'
- Corrected typo

Made the following updates in this preview of the [Metrics](http://file.ams.redhat.com/aecollin/JK-9743/Metrics/README.html) guide:

- Amended 'refer to' to 'see'
- Added ^ symbol to external links
- Ensured full name of product used on first occurrence in each module
- Changed 'Kafkacat' to 'Kcat'

### CODE REVIEW

File: docs/kafka/message-browsing-kafka/README.adoc [hunk 1]
```diff
@@ -143,40 +147,39 @@ Similarly, if a message is encoded (for example, in a format such as UTF-8 or Ba
...
+NOTE: If you have many partitions, you can filter the values shown in the *Partition* list by typing a value in the field.
 
 . To further refine the list of messages in the table, use the filter controls at the top of the *Messages* page.
 +
 --
 * To filter messages by topic partition and offset, perform the following actions:
 ... In the *Partition* field, select a topic partition.
-... In the drop-down menu that shows a default value of `Offset`, keep this default value.
-... In the *Offset* field, type an offset value.
+... Click the arrow next to the default value of `Latest messages` and select `Offset` from the list.
```
Comment (jbyrne-redhat) [inline docs/kafka/message-browsing-kafka/README.adoc:159]: Think we can just say "Click the arrow next to `Latest messages`..."

File: docs/kafka/message-browsing-kafka/README.adoc [hunk 2]
```diff
@@ -143,40 +147,39 @@ Similarly, if a message is encoded (for example, in a format such as UTF-8 or Ba
...
+... In the *Specify offset* field, type an offset value.
 ... To apply your filter settings, click the search (magnifying glass) icon.
 
 * To filter messages by topic partition and date and time, perform the following actions:
 ... In the *Partition* field, select a topic partition.
-... In the drop-down menu that shows a default value of `Offset`, change the value to `Timestamp`.
+... Click the arrow next to the default value of `Latest messages` and select `Timestamp` from the list.
```
Comment (jbyrne-redhat) [inline docs/kafka/message-browsing-kafka/README.adoc:165]: See https://github.com/redhat-developer/app-services-guides/pull/588/files#r986673369

File: docs/kafka/message-browsing-kafka/README.adoc [hunk 3]
```diff
@@ -143,40 +147,39 @@ Similarly, if a message is encoded (for example, in a format such as UTF-8 or Ba
...
 +
 Additional selection tools appear.
 ... Use the additional selection tools to set date and time values. Alternatively, type a date and time value in the format shown in the field.
 ... To apply your filter settings, click the search (magnifying glass) icon.
 
 * To filter messages by topic partition and epoch timestamp, perform the following actions:
 ... In the *Partition* field, select a topic partition.
-... In the drop-down menu that shows a default value of `Offset`, change the value to `Epoch timestamp`.
+... Click the arrow next to the default value of `Latest messages` and select `Epoch timestamp` from the list.
```
Comment (jbyrne-redhat) [inline docs/kafka/message-browsing-kafka/README.adoc:173]: See https://github.com/redhat-developer/app-services-guides/pull/588/files#r986673369

File: docs/kafka/message-browsing-kafka/README.adoc [hunk 4]
```diff
@@ -143,40 +147,39 @@ Similarly, if a message is encoded (for example, in a format such as UTF-8 or Ba
...
 ... In the *Epoch timestamp* field, type or paste an epoch timestamp value.
 +
 NOTE: You can easily convert a human-readable date and time to an epoch value using a https://www.epochconverter.com/[timestamp conversion tool^].
 ... To apply your filter settings, click the search (magnifying glass) icon.
 
 --
 +
-Based on your filter settings, the *Messages* page automatically reloads the list of messsages in the table.
+Based on your filter settings, the *Messages* page automatically reloads the list of messages in the table.
```
Comment (jbyrne-redhat) [inline docs/kafka/message-browsing-kafka/README.adoc:181]: :ok_hand:

**Comment by AedinC** [comment]
> Thanks for making these updates, @AedinC - they look good. Just a couple of small comments to consider.

Thanks for the feedback Jonh!

**Pull request merged**